.buildkite/scripts/steps/checks/alerts_as_data.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+echo --- Check Framework Alerts as Data Schema
+
+chmod 755 ./x-pack/plugins/alerting/common/alert_schema/scripts/generate_schemas.sh
+./x-pack/plugins/alerting/common/alert_schema/scripts/generate_schemas.sh
+chmod 644 ./x-pack/plugins/alerting/common/alert_schema/scripts/generate_schemas.sh
```

Comment:
CI check to ensure that changes to the alert schema are updated properly. I got a permissions error initially so added the `chmod` calls but maybe there is a better way to do this?